scheme/parsec-tpm/sanitize.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2024 Contributors to the Veraison project.
```

Comment:
I am wondering this generic implementation of string validation and checking for specific patterns is applicable more generically across entire Services `scheme` and could be moved to `services/common` ??

Reply:
ok, understood, thanks. Are these then coming from a spec reference for restriction in TPM specific scene setting?

Reply:
Not that I'm aware of (as I've said -- I'm not familiar with the specs). But this looks like it's designed specifically to catch "unreasonable" vendor/model names. From that point of view, the implementation looks reasonable. But I would not accept this as a good generic santizer (not do I think it's necesarily a good idea to have one).